be/src/exec/iceberg/iceberg_delete_builder.cpp

```diff
@@ -147,6 +147,7 @@ Status IcebergDeleteBuilder::build_parquet(const TIcebergDeleteFile& delete_file
         RETURN_IF_ERROR(status);
         RETURN_IF_ERROR(fill_skip_rowids(chunk));
     }
+    _skip_rows_ctx->deletion_bitmap = std::make_shared<DeletionBitmap>(_deletion_bitmap);
```

Comment:
If code not run here, the memory of _deletion_bitmap will be leak?

Reply:
Fixed